components/dashboard/src/hooks/use-analytics-tracking.ts

```diff
@@ -7,9 +7,17 @@
 import { useEffect } from "react";
 import { useHistory } from "react-router";
 import { trackButtonOrAnchor, trackPathChange } from "../Analytics";
+import { useUserLoader } from "./use-user-loader";
+import { useTheme } from "../theme-context";
+import { useOrbital } from "./use-orbital";
 
 export const useAnalyticsTracking = () => {
     const history = useHistory();
+    const { user, loading } = useUserLoader();
```

Comment:
Instead of this hook I think we should use the `const user = useCurrentUser()` hook, which just pulls the user from context as opposed to setting up a query to load the user. Below we can do our checks against the existence of that user and the onboardedTimestamp and not worry about the loading state (the user will be there once it's loaded if they're logged in).